Formula/gdcm.rb

```diff
@@ -25,7 +26,7 @@ class Gdcm < Formula
   depends_on "swig" => :build
   depends_on "openjpeg"
   depends_on "openssl@1.1"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
   depends_on "vtk@8.2"
```

Comment:
May be worth checking what features we lose by dropping VTK support. Fedora and AUR (and maybe FreeBSD, not sure of DSL they use) has disabled VTK. NixOS defines VTK as optional.

Reply:
Opened #107657 for this approach